sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtil.java

```diff
@@ -114,6 +115,22 @@ static void configureOtlpExporterBuilder(
     }
   }
 
+  static void configureOtlpAggregationTemporality(
+      ConfigProperties config, Consumer<AggregationTemporality> setAggregationTemporality) {
+    String temporalityStr = config.getString("otel.exporter.otlp.metrics.temporality");
+    if (temporalityStr == null) {
+      return;
+    }
+    AggregationTemporality temporality;
+    try {
+      temporality = AggregationTemporality.valueOf(temporalityStr);
```

Comment:
Should we do `toUpperCase`?

Reply:
I couldn't find any precedent for parsing config to enum values, but for the propagators which have a known set of config values all  in lower case, we don't convert to lower when checking equality.

Reply:
Yeah - so I expect users to generally want to pass these in as lower-case since enum values are generally lowercase. I guess `valueOf` will fail for that. How about just adding a switch instead then to be similar to the others?

Reply:
Actually noticed spec says case-insensitive. So I guess we would want `toUpperCase` to accept the lowercase versions

Reply:
Ah so it does. It appears to be the only env var that is to be interpreted with case insensitivity. Although, it is still experimental and subject to change. 

Pushed a commit with the update 👍.